app/src/main/java/br/org/cesar/discordtime/stickysessions/data/local/repository/UserLocalRepository.java

```diff
@@ -0,0 +1,33 @@
+package br.org.cesar.discordtime.stickysessions.data.local.repository;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import br.org.cesar.discordtime.stickysessions.domain.repository.UserRepository;
+
+public class UserLocalRepository implements UserRepository {
```

Comment:
I think it would be better if the `UserRepository` interface returns `Single` type from his methods, so you would need to decide anything about how a particular operation is being executed on UseCases.